"DockingLayoutException: null" when persistence file is in user.home

Windows 11
Java 21
modern docking v`SNAPSHOT-0.11`

```java
        var path = System.getProperty("user.home")+File.separator+"RobotOverlord"+File.separator+"ro3.layout";
        logger.debug(path);
        AppState.setPersistFile(new File(path));

        try {
            AppState.restore();
        } catch (DockingLayoutException e) {
            logger.error("Failed to restore docking layout.", e);
        }
```

Folder `C:\Users\aggra\RobotOverlord\` exists and is writable.  
On first run, MD creates the layout file.  
On second run, fails to load.

```
06:21:01.285 DEBUG C:\Users\aggra\RobotOverlord\ro3.layout
06:21:01.292 ERROR Failed to restore docking layout.
ModernDocking.exception.DockingLayoutException: null
	at modern_docking.api/ModernDocking.api.LayoutPersistenceAPI.loadApplicationLayoutFromFile(LayoutPersistenceAPI.java:147)
	at modern_docking.api/ModernDocking.api.AppStateAPI.restore(AppStateAPI.java:178)
	at modern_docking.single_app/ModernDocking.app.AppState.restore(AppState.java:104)
	at com.marginallyclever.robotoverlord/com.marginallyclever.ro3.apps.RO3Frame.createLayout(RO3Frame.java:312)
	at com.marginallyclever.robotoverlord/com.marginallyclever.ro3.apps.RO3Frame.<init>(RO3Frame.java:75)
	at com.marginallyclever.robotoverlord/com.marginallyclever.ro3.RO3.lambda$main$0(RO3.java:16)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:318)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.IllegalArgumentException: Illegal attribute index, 4, current START_ELEMENT has 4 attributes
	at com.fasterxml.aalto@1.3.2/com.fasterxml.aalto.stax.StreamReaderImpl.reportInvalidAttrIndex(StreamReaderImpl.java:1832)
	at com.fasterxml.aalto@1.3.2/com.fasterxml.aalto.stax.StreamReaderImpl.getAttributeValue(StreamReaderImpl.java:377)
	at modern_docking.api/ModernDocking.api.LayoutPersistenceAPI.readLayoutFromReader(LayoutPersistenceAPI.java:424)
	at modern_docking.api/ModernDocking.api.LayoutPersistenceAPI.loadApplicationLayoutFromFile(LayoutPersistenceAPI.java:134)
	... 19 common frames omitted```

--- a/docking-api/src/ModernDocking/api/LayoutPersistenceAPI.java
+++ b/docking-api/src/ModernDocking/api/LayoutPersistenceAPI.java
@@ -421,7 +421,7 @@ public class LayoutPersistenceAPI {
         String locStr = reader.getAttributeValue(1);
         String sizeStr = reader.getAttributeValue(2);
         int state = Integer.parseInt(reader.getAttributeValue(3));
-        String maximizedDockable = reader.getAttributeValue(4);
+        String maximizedDockable = reader.getAttributeCount() >= 5 ? reader.getAttributeValue(4) : null;
 
         Point location = new Point(Integer.parseInt(locStr.substring(0, locStr.indexOf(","))), Integer.parseInt(locStr.substring(locStr.indexOf(",") + 1)));
         Dimension size = new Dimension(Integer.parseInt(sizeStr.substring(0, sizeStr.indexOf(","))), Integer.parseInt(sizeStr.substring(sizeStr.indexOf(",") + 1)));

--- a/docking-single-app/build.gradle
+++ b/docking-single-app/build.gradle
@@ -27,6 +27,8 @@ base {
 
 dependencies {
 	implementation project(':docking-api')
+
+	implementation 'com.fasterxml:aalto-xml:1.3.1'
 }
 
 java {

